src/colp/components/usetype_version_comparison_report.py

```diff
@@ -0,0 +1,76 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import plotly.express as px
+from st_aggrid import AgGrid
+from src.constants import COLOR_SCHEME
+from abc import ABC
+import pdb
```

Comment:
Why are we importing pdb here?

Reply:
probably an oversight, should probably come out

Reply:
@SashaWeinstein are you going to take care of that? I can do it if need be and  this PR can be approved

Reply:
sure, I'll address now